bundles/org.openhab.binding.mqtt.homeassistant/pom.xml

```diff
@@ -40,5 +40,16 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.hubspot.jinjava</groupId>
+      <artifactId>jinjava</artifactId>
+      <version>2.7.2</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.re2j</groupId>
```

Comment:
Shouldn't this use the osgiified version too?

Reply:
The difference between an OSGi bundle a a non-OSGi bundle is only the content of the `MANIFEST.MF`. In the case of an OSGi bundle the manifest contains a list of imported and exported packages, allowing the OSGi framework to decide what is needed to get this bundle running. What is done in the osgiify repository is essentially unpacking the `.jar`, examining the classes and creating a list of needed and provided packages, putting that together in a new `MANIFEST.MF` and finally package it again in a `.jar`. So from the classes contained, the osgiified bundle and the original bundle are the same. The additional statements are not used in the maven build, so we can stick to the original version.

The feature-verification checks if all required packages are available at runtime (i.e. it could wire all import statements to bundles). It takes the framework bundles (the openHAB core runtime bundles) and all bundles declared in the `feature.xml` to do this and there the additional statements are needed, otherwise it wouldn't know what is provided by the jinjava bundle, so we need to use the osgiified version.

Reply:
Thanks for the details, this helped me to get a better understanding of how this fits together.